forms-flow-web-root-config/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - REACT_APP_KEYCLOAK_URL_REALM=${KEYCLOAK_URL_REALM:-forms-flow-ai}
       - REACT_APP_KEYCLOAK_URL=${KEYCLOAK_URL}
       - REACT_APP_APPLICATION_NAME=${APPLICATION_NAME:-formsflow.ai}
-      - REACT_APP_USER_ACCESS_PERMISSIONS=${USER_ACCESS_PERMISSIONS}
+      - REACT_APP_ENABLE_APPLICATION_ACCESS_PERMISSION_CHECK=${REACT_APP_ENABLE_APPLICATION_ACCESS_PERMISSION_CHECK:-false}
```

Comment:
please check the REACT_APP on the right side we need it or not?

Reply:
Updated